source/components/organisms/Step/Step.js

```diff
@@ -93,6 +93,8 @@ function Step({
     }
   };
   const inSubstep = currentPosition.level !== 0;
+  const inLastMainStep =
```

Comment:
A common way of writing these boolean variabels is to start the variabel with "is", so in this case isLastMainStep or isLastStep might cut it :)
Here is a short article for context on the matter 
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
I was just following the naming of the existing inSubstep variable. But sure, we can rename both of them to start with is instead, might be better. 

However I kind of agree with the top comment on that blog post, that just blindly prefixing things with is seems like a bad idea; since it can make things  sound like questions. Also, Hungarian notation is an anti-pattern and all that.

Reply:
Renamed both booleans to isSubstep and isLastMainStep, and did the corresponding refactoring.